README.md

```diff
@@ -62,6 +74,9 @@ Example:
 
 5. Build the whole infrastructure:
 
+The default setting is to activate the RabbitMQ first, then the IRI and the remote workers.
+Beware that the RabbitMQ must be the first one, otherwise the connection between the services can not be setup.
```

Comment:
Should be "set up" rather than "setup".
Check this: http://www.future-perfect.co.uk/grammar-tip/is-it-setup-set-up-or-set-up/

Reply:
Done. Thanks for the explanation reference.